tests/end-to-end/backup-restore-test/deploy/chart/backup-test/values.yaml

```diff
@@ -4,5 +4,5 @@ containerRegistry:
 nameOverride: backup-test
 
 image:
-  dir: develop/
-  tag: 164eb8ac
+  dir: pr/tests/backup-restore-test
```

Comment:
This breaks the convention for images naming we have now. Maybe split it into `dir`, `name` and `tag`?